css/style.css

```diff
@@ -0,0 +1,78 @@
+body {
+
+
+  background-color: black;
+  font-color: white;
+  font-family: serif;
+  font-size: 18px;
+  border-top: 5px solid #222222;
+  margin-top: 20px;
```

Comment:
@daniellasheva Just a tip, you could have also combined your two margin declarations into one shorthand declaration:
`margin: 20px 0 0 70px;`